CHANGELOG.md

```diff
@@ -4,6 +4,68 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Added `deploy voila` command to deploy Jupyter Voila notebooks. 
+
+### Changed
+- `deploy html` was refactored. Its behavior is described as 
+follows:
+
+### deploy html
```

Comment:
this feels like something that needs to be in our [docs](https://docs.posit.co/rsconnect-python/#static-snapshot-deployment), not just in the changelog.

Reply:
Will open a separate PR for Connect docs. Will leave the changes in the changelog for now, unless there's anything we should cut and move over to the docs.